components/telemetry-operator/main.go

```diff
@@ -65,8 +65,10 @@ var (
 	logFormat                  string
 	logLevel                   string
 	certDir                    string
-	allowedFilterPlugins       string
-	allowedOutputPlugins       string
+	supportedFilterPlugins     string
+	supportedOutputPlugins     string
+	deniedFilterPlugins        string
```

Comment:
I don't think we need an explicit list of denied filter and output plugins. All plugins that are not explicitly allowed, are denied.